web/src/pages/dashboard.tsx

```diff
@@ -121,6 +122,7 @@ const DashboardPage = (props: Props): ReactElement => {
             {LookupOperatingPhaseById(userData?.profileData.operatingPhase).displayFullCalendar && (
               <FilingsCalendar operateReferences={props.operateReferences} />
             )}
+            <HideableTasks />
```

Comment:
It feels like we could move the logic of checking `LookupOperatingPhaseById(...).displayHideableRoadmapTasks` from within the component to here, to follow the pattern of all the components above. Was there a reason to choose one over the other?